modules/cereal/1.3.2/overlay/BUILD

```diff
@@ -0,0 +1,20 @@
+load("@rules_foreign_cc//foreign_cc:defs.bzl", "cmake")
+
+package(default_visibility = ["//visibility:public"])
+
+filegroup(
+    name = "all",
+    srcs = glob(["**"]),
+)
+
+cmake(
```

Comment:
nit: it would be nicer to actually include a native BUILD file for this.

Reply:
done, use native cc_library.